### PR TITLE
Added plugin DisplayLayerProgress

### PR DESCRIPTION
**Pull request by OllisGit**
New plugin that shows the progres (percentage, layer-info) of the print on Printer-Display and on NavBar.

### CODE REVIEW

File: _plugins/DisplayLayerProgress.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  caption: Example of estimated time left
+- url: /assets/img/plugins/detailedprogress/example-eta.jpg
+  alt: Example of estimated time of completition
+  caption: Example of estimated time of completition
+
+featuredimage: /assets/img/plugins/detailedprogress/example-eta.jpg
+
+---
+
+TODO add screenshots
```
Comment (foosel) [inline _plugins/DisplayLayerProgress.md]: Please add the screenshots (they are missing from the commit) then remove that TODO ;)

**Comment by OllisGit** [comment]
thx, for review!

**Comment by OllisGit** [comment]
upps...pressed the wrong button.

**Comment by OllisGit** [comment]
hmm.. I am new in "OpenSource-Business", question: Should I close this PR after removing my TODOs, so that you can start the review again? How does it works?

**Comment by foosel** [comment]
It's fine like it is, I squashed the commits.

**Pull request merged**